index.html

```diff
@@ -33,5 +42,144 @@ <h3>I'm a software Developer, </h3>
         </ul>
       </div>
     </section>
+    <section>
+      <div class="section-two">
+        <h1>My Recent Works</h1>
+          <img class="first-divider" src="./final-images/Vector.png" alt="a divider">
+          <img class="yoga-image" src="./final-images/Img Placeholder.png" alt="a lady doing yoga">
+          <h2>Multi-Post Stories</h2>
+          <p>A daily selection of privately personalized reads; no accounts or sign-ups required.
+            has been the industry's standard dummy text ever since the 1500s, when an unknown printer took
+            a standard dummy text.
+          </p>
+          <ul class="stack-holder">
+            <li><a href="#">CSS</a></li>
```

Comment:
- [ ] **[OPTIONAL]** Great job on implementing the suggested changes! I have one optional suggestion for further improvement. It would be wonderful if you could consider centering your project cards. Currently, they are not centered, but you can achieve this by adding a flex container and centering the children using the "justify-content: center" property. This small adjustment will enhance the overall visual balance of the design.

![image](https://github.com/prince-rutagengwa/mobile-portfolio/assets/107952724/3e5321eb-0f5c-48e1-acea-8923b3986fa1)